### PR TITLE
Bugfix FXIOS-8547 [v125] Update Fonts related to Inactive Tabs to use FXFontStyles

### PR DESCRIPTION
**Pull request by darkCavalier11**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-8547)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/18986)

## :bulb: Description
Replaced with `FXFontStyles`

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [x] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)
Simulator Screenshot - iPhone 15 - 2024-03-05 at 22 02 05

### CODE REVIEW

File: firefox-ios/Client/Frontend/Browser/Tabs/Views/InactiveTabs/InactiveTabsCell.swift [hunk 1]
```diff
@@ -21,9 +20,7 @@ class InactiveTabsCell: UICollectionViewListCell, ReusableCell, ThemeApplicable
     private lazy var bottomSeparatorView: UIView = .build { _ in }
 
     private lazy var titleLabel: UILabel = .build { label in
-        label.font = DefaultDynamicFontHelper.preferredFont(
-            withTextStyle: .caption1,
-            size: UX.titleFontSize)
+        label.font = FXFontStyles.Regular.subheadline.scaledFont()
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Browser/Tabs/Views/InactiveTabs/InactiveTabsCell.swift:23]: need to confirm with @cwzilla on this change

Comment (cyndichin) [reply firefox-ios/Client/Frontend/Browser/Tabs/Views/InactiveTabs/InactiveTabsCell.swift:23]: Discussed with @cwzilla, can we update this to use `.body` to be in sync with our fonts we used in our list items (i.e. settings, bookmarks)

Comment (darkCavalier11) [reply firefox-ios/Client/Frontend/Browser/Tabs/Views/InactiveTabs/InactiveTabsCell.swift]: done!

Comment (cyndichin) [reply firefox-ios/Client/Frontend/Browser/Tabs/Views/InactiveTabs/InactiveTabsCell.swift]: awesome, thank you @darkCavalier11 !

**Review by cyndichin — APPROVED** [review]
looks great, thank you for all the font adjustments @darkCavalier11!

**Pull request merged**